src/components/ListingDetail/ListingDetail.js

```diff
@@ -25,6 +56,15 @@ export default ({
       const id = this.getIdByPathname()
 
       return this.getHrefFromId(id)
+    },
+
+    imageURL() {
+      let medias = this.listing.medias
+      medias.forEach((obj) => {
```

Comment:
What do you think about rename this variable from `obj` to `media`?
Because when I read this code, in the first time, I understanding that mean each media is an element of medias.

Reply:
True, I've put `obj` cause medias is an Array of objects, but media sounds better.